test/service/test_tdr.py

```diff
@@ -89,8 +89,10 @@ def test(dataset: BigQueryDataset):
             test(BigQueryDataset('test-project', 'name', False))
 
     def test_tdr_dataset_config(self):
-        dataset = config.tdr_bigquery_dataset
-        self.assertNotEqual(dataset.is_snapshot, dataset.name.startswith('datarepo_'))
+        from deployments.sandbox.environment import env as sandbox_env
+        with mock.patch.object(type(config), 'tdr_target', sandbox_env()['AZUL_TDR_TARGET']):
```

Comment:
I don't think this is how you patch a property. This only passes because AZUL_TDR_TARGET is already set on Travis.

I'd like to see a drop! commit that patches this to the wrong value causing the assertion below to fail. That's proof that patch works.

Reply:
You could also simply patch the environment variable.